x-pack/legacy/plugins/infra/public/pages/logs/stream/page_header.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { injectI18n, InjectedIntl } from '@kbn/i18n/react';
+import React from 'react';
+
+import { UICapabilities } from 'ui/capabilities';
+import { injectUICapabilities } from 'ui/capabilities/react';
+import { DocumentTitle } from '../../../components/document_title';
+import { Header } from '../../../components/header';
+
+interface StreamPageHeaderProps {
+  intl: InjectedIntl;
+  uiCapabilities: UICapabilities;
+}
+
+export const StreamPageHeader = injectUICapabilities(
+  injectI18n((props: StreamPageHeaderProps) => {
```

Comment:
We're not supposed to be using `injectI18N` anymore: https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#in-reactjs

Existing files with `inject` are no big deal IMO, but ion new code let's replace with `i18n.translate` as recommended.

Reply:
Will fix 👍